backends/arm/test/ops/test_acos.py

```diff
@@ -105,10 +104,7 @@ def test_acos_vgf_FP(test_data: Tuple):
         tosa_version="TOSA-1.0+FP",
         run_on_vulkan_runtime=True,
     )
-    try:
-        pipeline.run()
-    except FileNotFoundError as e:
-        pytest.skip(f"VKML executor_runner not found - not built - skip {e}")
+    pipeline.run()
```

Comment:
enabled by this PR?

Reply:
no this was just moving the try/except to a central location